web/themes/custom/novel/templates/fields/field--eventinstance--event-categories--list-teaser.html.twig

```diff
@@ -0,0 +1,3 @@
+{% include 'field--field-categories.html.twig' with {
+  item_attributes: create_attribute().addClass('event-list-item__tag')
```

Comment:
Doesnt `attributes` already exist on the items?

Reply:
You are totally right. I replaced this with proper `item.attributes` in [b294c0d](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/665/commits/b294c0d39fc5346b7f04f1b35c81e5976004d408). Much better!